database/migrations/1732127232754-create-review-entity.ts

```diff
@@ -0,0 +1,96 @@
+import {
+  MigrationInterface,
+  QueryRunner,
+  Table,
+  TableForeignKey,
+} from 'typeorm';
+
+export class CreateReviewEntity1732127232754 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.createTable(
+      new Table({
+        name: 'review',
+        columns: [
+          {
+            name: 'id',
+            type: 'uuid',
+            isPrimary: true,
+            generationStrategy: 'uuid',
+            default: 'uuid_generate_v4()',
+          },
+          {
+            name: 'created_at',
+            type: 'timestamp',
+            isNullable: false,
+            default: 'now()',
+          },
+          {
+            name: 'updated_at',
+            type: 'timestamp',
+            isNullable: false,
+            default: 'now()',
+          },
+          {
+            name: 'content',
+            type: 'text',
+            isNullable: true,
+          },
+          {
+            name: 'rating',
+            type: 'int',
+            isNullable: false,
+          },
+          {
+            name: 'account_id',
+            type: 'uuid',
+            isNullable: true,
+          },
+          {
+            name: 'accommodation_id',
+            type: 'uuid',
+            isNullable: true,
+          },
+          {
+            name: 'booking_id',
+            type: 'uuid',
+            isNullable: true,
+          },
+        ],
+      }),
+    );
+
+    await queryRunner.createForeignKey(
+      'review',
+      new TableForeignKey({
+        columnNames: ['account_id'],
+        referencedColumnNames: ['id'],
+        referencedTableName: 'account',
+        onDelete: 'SET NULL',
+      }),
+    );
+
+    await queryRunner.createForeignKey(
+      'review',
+      new TableForeignKey({
+        columnNames: ['accommodation_id'],
+        referencedColumnNames: ['id'],
+        referencedTableName: 'accommodation',
+        onDelete: 'SET NULL',
```

Comment:
It's more logical to cascade delete reviews. Who needs a review that doesn't have accommodation?